### ISSUE TITLE
Support New Values for ELBv2 Listener Actions

### ISSUE THREAD

**Issue by melusom** [awslabs/cfn-python-lint#573]
*cfn-lint version: 0.11.1*

*AWS::ElasticLoadBalancingV2::Listener DefaultActions does not support new values*

[`Elastic Load Balancing Listener Action`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-listener-defaultactions.html) does not conform to the latest documentation and supported values. Below is a sample valid snippet:

```yaml
  rLoadBalancerListenerHttp:
    Type: AWS::ElasticLoadBalancingV2::Listener
    Properties:
      LoadBalancerArn: !Ref rLoadBalancer
      Port: 80
      Protocol: 'HTTP'
      DefaultActions:
        - Type: redirect
          RedirectConfig:
            - StatusCode: HTTP_301
              Port: '443'
              Protocol: HTTPS
```
`cfn-lint` returns these errors, both of which are false-positives:

* [cfn-lint] E3003:Property TargetGroupArn missing at Resources/rLoadBalancerListenerHttp/Properties/DefaultActions/0
* [cfn-lint] E3002:Invalid Property Resources/rLoadBalancerListenerHttp/Properties/DefaultActions/0/RedirectConfig

Per the linked documentation above, `TargetGroupArn` is only required on `Type: forward`, and `RedirectConfig` is not only valid but a required property when using `Type: redirect`

**Comment by kddejong** [comment]
@melusom are you sure you are on v0.11.1?  Looking back through the spec history it looks like we picked up support for the DefaultActions at version v0.10.0.  This is the only error I get when running you example code through the most recent version (v0.12.1 as of today).

```
E3002 Expecting an object at Resources/rLoadBalancerListenerHttp/Properties/DefaultActions/0/RedirectConfig
test1.yaml:11:11
```

**Comment by melusom** [comment]
@kddejong typo recognized/corrected in line 8 of my snippet. I had run `cfn-lint --version` prior to posting the issue, but I'm unable to reproduce it now and am still on v0.11.1. Since I can no longer reproduce it I will close this issue.
